sunpy/spectra/spectrogram.py

```diff
@@ -274,28 +274,28 @@ class Spectrogram(Parent):
 
     Attributes
     ----------
-    data : `~numpy.ndarray`
+    `~data` : `~numpy.ndarray`
```

Comment:
this needs updating. The following conventions should be followed for docstrings:

1. Types (both built-in and things like `numpy.ndarray`) should be in single  backticks and optionally have a `~` in front to only display the name of the class and not the whole thing, i.e `ndarray` over `numpy.ndarray`. (So built in types should not have a tilde).
2. Variable names that are not importable (like arguments to functions) should be in double backticks i.e:
```
``data``
```